src/assets/css/main.css

```diff
@@ -155,3 +155,14 @@ Transition styles for <transition />
   opacity: 0;
   transform: translateY(-20px);
 }
+
+li.related-item:nth-child(even){
+  background: rgb(250, 250, 252)
+}
+
+td.related-item{padding:0;}
+
+li.related-item{
+  padding: 12px;
+  border-left: rgb(239, 239, 245)
+}
```

Comment:
These should be added in tailwind directly on the component. You can dynamically assign classes via `:class=""`

Reply:
can tailwind gray replace this specific strip tone of rgb(250, 250, 252)?

Reply:
I don't think you need the exact value. The closest tailwind color should match. But if you wanted to: https://tailwindcss.com/docs/border-color#arbitrary-values